CONTRIBUTING.md

```diff
@@ -61,80 +65,77 @@ We also like the Recurse Center's [social rules](https://www.recurse.com/manual#
-  updates?
+* `Pull Request Docs`: Did you make any necessary documentation updates?
 
-* License: by submitting a PR, you're offering your
-  changes under this project's license.
+* License: by submitting a PR, you're offering your changes under this
+  project's license.
 
 
 
 What to put in a PR
 -----------------------
 
 Each PR should, as much as possible, address just one issue and be
-self-contained. If you have ten small, unrelated changes, then go
-ahead and submit ten PRs – it's much easier to review ten small
-changes than one big change with them all mixed together, and this way
-if there's some problem with one of the changes it won't hold up all
-the others.
-
-If you're uncertain about whether a change is a good idea and want
-some feedback before putting time into it, feel free to ask in an
-issue or in the discussions tab.  If you have a partial change that you want
-to get feedback on, feel free to submit it as a PR. (In this case it's
-traditional to start the PR title with `[WIP]`, for "work in
-progress".)
-
-When you are submitting your PR, you can include ``Closes #123``,
-``Fixes: #123`` or [some variation](https://help.github.com/en/articles/closing-issues-using-keywords) in either your commit message or the PR description, in order to
-automatically close the referenced issue when the PR is merged.
-This keeps us closer to the desired state where each open issue reflects some
-work that still needs to be done.
+self-contained. If you have ten small, unrelated changes, then go ahead and
+submit ten PRs – it's much easier to review ten small changes than one big
+change with them all mixed together, and this way if there's some problem with
+one of the changes it won't hold up all the others.
+
+If you're uncertain about whether a change is a good idea and want some
+feedback before putting time into it, feel free to ask in an issue or in the
+discussions tab. If you have a partial change that you want to get feedback on,
+feel free to submit it as a PR. (In this case it's traditional to start the PR
+title with `[WIP]`, for "work in progress".)
+
+When you are submitting your PR, you can include ``Closes #123``, ``Fixes:
+#123`` or [some variation](https://help.github.com/en/articles/closing-issues-using-keywords)
+in either your commit message or the PR description, in order to automatically
+close the referenced issue when the PR is merged.  This keeps us closer to the
+desired state where each open issue reflects some work that still needs to be
+done.
 
 
 Tests
 -----
 
-We use [pytest](https://pytest.org/) for testing. To run the tests
-locally, you should run:
+We use [pytest](https://pytest.org/) for testing. To run the tests locally, you
+should run:
 
 ```shell
-cd path/to/repo/checkout/
-pip install -r ../checkout  # possibly using a virtualenv
-pytest src
+$ cd path/to/repo/checkout
+$ python -m venv venv
+$ source venv/bin/activate
+$ pip install -r test-requirements.txt  # installs e.g. pytest
+$ pytest
```

Comment:
I agree with upgrading the commands here, but adding `$` in front of it means you can't easily use github's copy codeblock functionality (icon in top right when markdown is rendered)

Reply:
With #112 being merged now as well, we can update this to day `uv sync` instead of `pip install -r`

Reply:
valid point. First time using "uv sync" for me. It seems to also create the virtual environment. Not sure if I updated correctly but it seems to work for me.

Reply:
while looking for the uv installation, I found, that [material](https://squidfunk.github.io/mkdocs-material/) seems to allow copying `console` content with leading `$`.
See [source](https://github.com/astral-sh/uv/blob/main/docs/getting-started/installation.md) vs [build](https://docs.astral.sh/uv/getting-started/installation/#standalone-installer)